src/ORM/Translatable/TranslatableSubscriber.php

```diff
@@ -68,11 +68,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($this->isTranslatable($classMetadata)) {
+        if ($this->requiresTranslatableMapping($classMetadata)) {
```

Comment:
The previous `isTranslatable` name was more understandable IMHO, and also in a way more accurate as `requires` could make think of a state awareness. WDYT?